main/main.cpp

```diff
@@ -427,6 +427,7 @@ void Main::print_help(const char *p_binary) {
 #endif
 	OS::get_singleton()->print("  --frame-delay <ms>                Simulate high CPU load (delay each frame by <ms> milliseconds).\n");
 	OS::get_singleton()->print("  --time-scale <scale>              Force time scale (higher values are faster, 1.0 is normal speed).\n");
+	OS::get_singleton()->print("  --seed <seed>                     Specify a fixed seed for better reproducibility.\n");
```

Comment:
Only the global RandomNumberGenerator instance is affected by this CLI argument, not local instances that may be created in game logic:

```suggestion
	OS::get_singleton()->print("  --seed <seed>                     Specify a fixed random number generation seed for the global RandomNumberGenerator instance for better reproducibility.\n");
```

Reply:
> Only the global RandomNumberGenerator instance is affected by this CLI argument

I am currently investigating ways to change `randomize()` to transparently [split its new seed from the global instance](https://jax.readthedocs.io/en/latest/jax.random.html#prng-keys)